package.json

```diff
@@ -8,8 +8,8 @@
     "rbac:setup": "./build/rbac-setup.sh",
     "check": "prettier --check .",
     "check:fix": "prettier --write .",
-    "test": "npm run test:clean-reports && npm run test:headless && npm run test:merge-reports",
```

Comment:
I think we need to keep the merge-reports.  Removing it may affect the output expected from the canary.

Reply:
Readding the `merge-reports`. I removed it at first since we call it within the `start-tests.sh` script.